src/main/java/org/traccar/protocol/TrvProtocolDecoder.java

```diff
@@ -181,6 +181,8 @@ protected Object decode(
             } else if (type.equals("AP14") && !id.equals("IW")) {
                 channel.writeAndFlush(new NetworkMessage(responseHeader + ",0.000,0.000#", remoteAddress));
             } else if (!type.equals("AP12") && !type.equals("AP14")
+                    && !type.equals("AP33") && !type.equals("AP34")
+                    && !type.equals("AP84") && !type.equals("AP85")
```

Comment:
I think we should do something like `!Set.of("AP12", ..., "AP85").includes(type)`.

Reply:
if (channel != null) {
    String responseHeader = id + (char) (type.charAt(0) + 1) + type.substring(1);
    
    Set<String> excludedTypes = new HashSet<>(Arrays.asList("AP12", "AP14", "AP33", "AP34", "AP84", "AP85"));

    if (type.equals("AP00") && id.equals("IW")) {
        // Zeitformatierung nur einmal erstellen
        String time = new SimpleDateFormat("yyyyMMddHHmmss").format(new Date());
        channel.writeAndFlush(new NetworkMessage(responseHeader + "," + time + ",0#", remoteAddress));
    } else if (type.equals("AP14") && !id.equals("IW")) {
        channel.writeAndFlush(new NetworkMessage(responseHeader + ",0.000,0.000#", remoteAddress));
    } else if (!excludedTypes.contains(type) && !sentence.substring(responseHeader.length() + 1).matches("^\\d{6}$")) {
        channel.writeAndFlush(new NetworkMessage(responseHeader + "#", remoteAddress));
    }
}

Reply:
Why can't you use `Set.of` inline, like I suggested?